pages/common/texdoc.md

```diff
@@ -0,0 +1,16 @@
+# texdoc
+
+> Search for appropriate documentation for (La)TeX commands or packages.
+> More information: <https://texdoc.org/index.html>.
+
+- Open first search result in the default PDF viewer:
+
+`texdoc {{search}}`
+
+- List best search result:
+
+`texdoc --list {{search}}`
+
+- Open full documentation of texdoc:
+
+`texdoc texdoc`
```

Comment:
I know you specified "texdoc" in the descript but variables arguments we usually denote as such for clarification.
```suggestion
`texdoc {{texdoc}}`
```